committer-tools/verify_license.py

```diff
@@ -74,12 +75,20 @@ def get_license_deps(license_text):
     return set(LICENSE_DEP_PATTERN.findall(license_text))
 
 def main():
+    # Argument parser
+    parser = argparse.ArgumentParser(description="Whether to skip executing ReleaseTarGz.")
+    parser.add_argument('--skipReleaseTarGz', type=str, default=False, help='Whether to skip ReleaseTarGz (true/false)')
```

Comment:
I would suggest a flag name like "rebuild" or similar. 

Also, this is not the correct way to configure a boolean argument. You should do something like:

```python
parser.add_argument("--my-flag", action="store_true", help="Help text")
```

https://docs.python.org/3/library/argparse.html#action

Reply:
> I would suggest a flag name like "rebuild" or similar.

The script should build the distribution if the flag is unset, as it shouldn't assume a distribution already exists. Perhaps the flag could be named skipBuild?